README.md

```diff
@@ -474,6 +474,12 @@ console.log(fileBox.remoteSize)
 1. Serializable
 1. Can be Transfered from server to server, server to browser.
 
+## Environments
+
+Environment variables can be used to control some behavior.
+
+- `FILEBOX_HTTP_TIMEOUT` [default=5000] Socket idle timeout when FileBox downloads data from URL. For example, when the network is temporarily interrupted, the request will be considered as failed after waiting for a specified time.
```

Comment:
Why the default is 5 seconds? I feel it's too short.

Reply:
I think the underlying socket connection has been broken when the network fails. After about 5 seconds, even if the network is restored, the socket will still not resume work. If set too long timeout here, will just wait for the timeout meaninglessly.

Reply:
>Sets the socket to timeout after timeout milliseconds of inactivity on the socket. By default net.Socket do not have a timeout.
>
>When an idle timeout is triggered the socket will receive a ['timeout'](https://nodejs.org/docs/latest-v18.x/api/net.html#event-timeout) event but the connection will not be severed. The user must manually call [socket.end()](https://nodejs.org/docs/latest-v18.x/api/net.html#socketenddata-encoding-callback) or [socket.destroy()](https://nodejs.org/docs/latest-v18.x/api/net.html#socketdestroyerror) to end the connection.

- https://nodejs.org/docs/latest-v18.x/api/http.html#requestsettimeouttimeout-callback
- https://nodejs.org/docs/latest-v18.x/api/net.html#socketsettimeouttimeout-callback

Reply:
The code here I think it’s a stream instead of a socket?